docs/src/main/paradox/buildtools/gradle.md

```diff
@@ -13,25 +13,14 @@ By default, both client and server are generated and Java or Scala is autodetect
 
 ### Installation
 
-**The Gradle plugin is not yet deployed to plugins.gradle.org.**
-
-For now, you will you need to build the plugin yourself and deploy to your local Maven repository.
-Unfortunately, the [Gradle Plugin Publishing](https://docs.gradle.org/current/userguide/publishing_gradle_plugins.html) plugin
-does not appear to support publishing to repositories that require authentication (e.g. Artifactory).
-
-If you check out the source code from our [git repo](https://github.com/apache/incubator-pekko-grpc)
-or [download](https://pekko.apache.org/download.html) a source release, you can change directory to the
-`gradle-plugin` directory and run this Gradle command.
-
-```shell
```

Comment:
Could you move these local build instructions to the README build from source section if you don't think we could keep them here?

Reply:
Wouldn't `CONTRIBUTING.md` be a better spot for these instructions? As a user there really isn't a need to build the project manually, only if you are developing/doing changes to the plugin

Reply:
Maybe. It's just that it's useful to have a shortened version of the build instructions for release voters. At a minimum, we need to remove the Gradle link in the README because it will become a deal link.

Reply:
Okay so I just added info to `README.md` and fixed the link, I think it should be fine now.